spec/features/bars/users_views_bar_details_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "non-user views a bar's details" do
+
+  let(:bar1) { Bar.create(name: "Happy Hour Lasagna", description: "Where are we at?") }
```

Comment:
Make this a `let!(...)` instead of a `let(...)` - that'll allow you to *not* need to call `bar1` below! Adding the bang forces the object to be created right away at the start of each scenario, whether or not it gets called later.